src/descanso/response_transofrmers.py

```diff
@@ -82,8 +84,8 @@ def transform_response(
         fields: dict[str, Any],
     ) -> HttpResponse:
         if (
```

Comment:
what if `codes=None, except_codes=[201]`?
do we expect error on status code `404`?